res/database/Default/database_ads.xml

```diff
@@ -1615,19 +1617,19 @@ Available in any well-stocked drugstore.</en>
 			</title>
 			<description>
 				<de>3D Scanner und 3D Drucker in einem. Denken Sie an die Möglichkeiten! Verwandeln Sie eine Metallvorlage in ein Plastik-Original. Doublie-Kate von "Life is Groats"!</de>
-				<en>3D scanner and 3D printer in one. Think about the possibilities! Transform a metal template into a plastic original. Doublie-Kate of "Life is Groats"!</en>
+				<en>3D scanner and 3D printer in one. Think of the possibilities! Turn a metal template into a plastic original. Doublie-Kate from "Life is Groats"!</en>
 			</description>
 			<conditions min_audience="28.65" min_image="0" target_group="0"/>
 			<data infomercial="1" quality="20" repetitions="2" duration="4" profit="156" penalty="309" infomercial_profit="35" fix_infomercial_profit="1"/>
 		</ad>
 		<ad id="speedminister-ad-Doerrobst" creator="8605" created_by="SpeedMinister">
 			<title>
 				<de>Dörrobst Toni</de>
-				<en>Desiccation Dave</en>
+				<en>Dried Fruit Dave</en>
```

Comment:
Dave's Dried Fruit

Reply:
yeah many cases like that, we could apply some variety

Reply:
I changed my mind. Leave it as it is.